test/Generics/unbound.swift

```diff
@@ -137,3 +137,19 @@ do {
   assertExactType(of: x1, is: Generic<Int, Bool>.self)
   assertExactType(of: x2, is: Generic<Int, Bool>.self)
 }
+
+// https://github.com/apple/swift/issues/60922
+enum T<E> {
+  // expected-note@-1{{generic type 'T' declared here}}
+  case a(E)
+  case b([E])
```

Comment:
The cases can me omitted; they aren’t involved anywhere in these tests or the code paths being tested.